lib/manager/nuget/extract.ts

```diff
@@ -78,6 +80,40 @@ export async function extractPackageFile(
     config.localDir
   );
 
+  if (packageFile.endsWith('.config/dotnet-tools.json')) {
```

Comment:
This will mean that users will be unable to use alternative filenames, even if they configure `fileMatch` accordingly. Maybe we'd be better to make this a new manager called `dotnet-tools`?